common/src/main/resources/messages/stack-status-messages.yaml

```diff
@@ -1,4 +1,5 @@
 'cluster-proxy.ccm.endpoint-unavailable': 'The Control Plane was not able to establish the connection with the gateway instance. This means that the reverse SSH tunnel (autossh process) running on this instance could not connect to the Cloudera server. Please check your connection and proxy settings and make sure the instance can reach *.ccm.cdp.cloudera.com. Refer to Cloudera documentation at https://docs.cloudera.com/management-console/cloud/connection-to-private-subnets/topics/mc-ccm-troubleshooting.html'
 'cluster-proxy.ccmv2.endpoint-unavailable': 'The Control Plane was not able to establish the connection with the gateway instance. This could be caused by the Jumpgate agent running on this instance not being able to connect to the Cloudera server. Please check your connection and proxy settings and make sure the instance can reach *.ccm.cdp.cloudera.com Please check your instance on the cloud provider side if it is up and running. Restart it if it could not start up properly.'
+'cluster-proxy.ccmv2.2.no-route-found': 'The Control Plane was not able to establish the connection with the gateway instance. This could be caused by the Jumpgate agent running on this instance not being able to connect to the Cloudera server. Please check your connection and proxy settings and make sure the instance can reach *.ccm.cdp.cloudera.com Please check your instance on the cloud provider side if it is up and running. Restart it if it could not start up properly.'
```

Comment:
this somehow sneaked in

Reply:
just investigated a problem this is why it is here